packages/cdktf-cli/src/lib/checkpoint.ts

```diff
@@ -142,7 +142,7 @@ information on how to disable it.`
 
 export async function ReportRequest(reportParams: ReportParams): Promise<void> {
   // we won't report when checkpoint is disabled.
-  if (CHECKPOINT_DISABLE) {
+  if (CHECKPOINT_DISABLE()) {
```

Comment:
Why does this need to be a function now? just curious

Reply:
I'll take this out, as it doesn't make sense in this PR, but I was a bit confused about this test working:

https://github.com/hashicorp/terraform-cdk/blob/e05a7c999cd38d20553f3c9daac26fd4a1f47da2/packages/cdktf-cli/src/test/checkpoint.test.ts#L36

It does seem to succeed, but I'm not sure why :/ since the process.env is read and set in this variable, and doesn't get re-read later.